### ISSUE TITLE
do back/forward ports (97)

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#7400]
https://github.com/borgbackup/borg/issues?q=label%3Aforwardport%2Fmaster%2Cbackport%2F1.1-maint%2Cbackport%2F1.2-maint

do 2 simple/small or 1 mid-sized/mid-complex forward/backport from there.

---

:moneybag: [there is a bounty for this](https://app.bountysource.com/issues/117548137-do-back-forward-ports-97)

**Comment by nakehealth** [comment]
how resolution?

**Comment by ThomasWaldmann** [comment]
@nakehealth can you rephrase your question, it's unclear to me what exactly you are asking.

**Comment by ThomasWaldmann** [comment]
- forward-port #7435 to master: #7442
- backport #7463: #7464

**Comment by ThomasWaldmann** [comment]
this is done.
